source/includes/_fiat_to_crypto_flow.md

```diff
@@ -13,7 +13,10 @@ curl -X POST 'https://api.onramp.ltd/exchange/fiat/crypto/invoice/new' \
 -d '{
     "fiat_amount": 3000,
     "fiat_currency": "EUR",
-    "callback_url": "https://www.example.com",
+    "payment_ack_url": "https://www.example.com/ack",
```

Comment:
@ivan-mudrak do we have then one or two callbacks?

Reply:
One, `callback_url` is intended to send status updates but it is not yet implemented. Shall I remove it then to avoid confusion?